src/checkers.js

```diff
@@ -413,6 +415,24 @@ function getCheckers(disabled) {
     }, {type}, disabled);
   }
 
+  function lessThanCheckGetter(min) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['lessThanCheckGetter' is defined but never used.](https://www.codacy.com/app/kentcdodds/api-check/file/738631697/issues/source?bid=2426116&fileBranchId=2432553#l418)